core/src/test/java/cucumber/api/TestStepTest.java

```diff
@@ -89,9 +89,12 @@ public void result_is_pending_when_step_definition_throws_pending_exception() th
 
     @Test
     public void step_execution_time_is_measured() throws Throwable {
-        Long duration = new Long(1234);
+        Long duration = 1234L;
         TestStep step = new PickleTestStep("uri", mock(PickleStep.class), definitionMatch);
-        when(bus.getTime()).thenReturn(0l, 1234l);
+
+        when(bus.getTime())
+            .thenReturn(0L)
+            .thenReturn(1234L);
 
```

Comment:
The solution was a neat little trick in the mockito API that allows calls be be chained. The first invocation will return the result of the first `thenReturn` the second that of the second and so on.